py3status/modules/getjson.py

```diff
@@ -68,28 +65,17 @@ class Py3status:
     def getjson(self):
         """
         """
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout),
-        }
-
         try:
-            resp = self.py3.request(self.url, timeout=self.timeout)
-            status = resp.status_code == 200
-            resp = resp.json()
+            getjson = self.py3.request(self.url, timeout=self.timeout).json()
+            getjson = self.py3.flatten_dict(
+                getjson, delimiter=self.delimiter, intermediates=True)
```

Comment:
imho this is ugly because you are between two approaches
```# I like this
function(
    params1, param2, param3
)

# sometimes this is great too
function(params1,
         param2,
         param3)


# this is bastard child that should not live
function(
    params1, param2, param3)
```